x-pack/plugins/lens/public/xy_visualization/visualization.tsx

```diff
@@ -167,16 +181,97 @@ export const getXyVisualization = ({
             position: Position.Top,
             seriesType: defaultSeriesType,
             showGridlines: false,
+            layerType: layerTypes.DATA,
           },
         ],
       }
     );
   },
 
+  getLayerType(layerId, state) {
+    return state?.layers.find(({ layerId: id }) => id === layerId)?.layerType;
+  },
+
+  getLayerTypes(state, frame) {
```

Comment:
I am not a fan of this name given that we have `getLayerType` that returns only type... I am thinking `getLayerTypesParams` or `getLayerTypesGroups`? Not ideal either :(